src/Acuminator/Acuminator.Analyzers/StaticAnalysis/DacExtensionDefaultAttribute/DacExtensionDefaultAttributeAnalyzer.cs

```diff
@@ -142,7 +143,7 @@ public static async Task<Location> GetAttributeLocationAsync(AttributeData attri
 
 			attributeSyntaxNode = await attribute.ApplicationSyntaxReference.GetSyntaxAsync(cancellationToken)
```

Comment:
Join declaration and assignment.
Use `ConfigureAwait(false)` - read the code style, for Christ's sake.